handler/src/main/java/io/netty/handler/ssl/JdkSslServerContext.java

```diff
@@ -65,6 +66,21 @@ public JdkSslServerContext(File certChainFile, File keyFile, String keyPassword)
                 JdkDefaultApplicationProtocolNegotiator.INSTANCE, 0, 0);
     }
 
+    /**
+     * Creates a new instance.
+     *
+     * @param certChainFile an X.509 certificate chain file in PEM format
+     * @param keyFile a PKCS#8 private key file in PEM format
+     * @param keyPassword the password of the {@code keyFile}.
+     *                    {@code null} if it's not password-protected.
+     * @param keyStore the used keystore type
+     */
+    JdkSslServerContext(File certChainFile, File keyFile,
```

Comment:
It looks like these new package-private constructors should be removed, as they have no callers. It seems SslContext is the only caller and it uses the "complete" constructor.